api/mock/store.go

```diff
@@ -55,7 +56,7 @@ func (s *StorageService) DeleteAttachment(accountID int, attachmentID int) error
 }
 
 // CreateOrUpdateSession creates a new session record in the db
-func (s *StorageService) CreateOrUpdateSession(accountID int, sessionKey string, expirationDate time.Time) error {
+func (s *StorageService) CreateOrUpdateSession(accountID int, sessionKey string, sessionIndex sql.NullString, expirationduration time.Duration) error {
```

Comment:
Now that we have `ExtendAndFetchSessionAccount`, can this just be `CreateSession` instead of `CreateOrUpdateSession` (in name and in implementation)?

Reply:
Yes, I think I will break it out so that it is more explicit when someone logs into an existing session.